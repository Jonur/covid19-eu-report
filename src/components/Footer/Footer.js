import React from 'react';
import s from './Footer.module.scss';

const Footer = () => (
  <footer className={s.globalFooter}>
    &copy;{`${new Date().getFullYear()} - `}
    <a
      rel="noopener noreferrer"
      target="_blank"
      href="https://github.com/Jonur"
      alt="Jonur on Github"
    >
      Jonur
    </a>
  </footer>
);

export default Footer;
