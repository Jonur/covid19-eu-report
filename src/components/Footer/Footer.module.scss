.globalFooter {
  font-size: 0.875rem;
  text-align: center;
  justify-self: flex-end;
  height: 2rem;
  line-height: 2rem;
}
