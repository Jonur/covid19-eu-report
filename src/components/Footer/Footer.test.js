import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
  test('renders the Footer component correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
