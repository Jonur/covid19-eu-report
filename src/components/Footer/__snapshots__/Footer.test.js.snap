// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the Footer component correctly 1`] = `
<div>
  <footer
    class="globalFooter"
  >
    Â©
    2020 - 
    <a
      alt="Jonur on Github"
      class="link"
      href="https://github.com/Jonur"
      rel="noopener noreferrer"
      target="_blank"
    >
      Jonur
    </a>
     - 
    <a
      alt="Application Privacy Policy"
      class="link"
      href="https://github.com/Jonur/covid19-eu-report/blob/master/PrivacyPolicy.md"
      rel="noopener noreferrer"
      target="_blank"
    >
      Privacy Policy
    </a>
  </footer>
</div>
`;
